src/emailLogin/runMigration.js

```diff
@@ -10,6 +10,24 @@ const ISOMER_USERS = ['isomeradmin', 'rc-davis', 'lamkeewei', 'pallani', 'LoneRi
 // name of repo
 const REPO = process.argv[2];
 
+const writeMigrationInfoToRecords = async (site, contributorRecord, repoRecord, insertRecord) => {
+  try {
+    // write repo information and queries run to file
+    if (!fs.existsSync('./emailMigrationData')) {
+      fs.mkdirSync('./emailMigrationData');
+    }
+    const dirPath = `./emailMigrationData/${site}`;
+    if (!fs.existsSync(dirPath)) {
+      fs.mkdirSync(dirPath);
+    }
+    fs.writeFileSync(`${dirPath}/contributors.txt`, contributorRecord);
```

Comment:
Are we storing these logs in the cloud?

Reply:
Nope, this is just for our own records in case we need to revert or something - I think we also might not want to push user data, so it'll be just to keep track of migrations locally

Reply:
nono cloud like our private notion/our google doc hahah

Reply:
Will push to a private folder in our drive!

Reply:
actually if this is required, why not just commit to git rather than ignoring it?